torch_geometric/nn/conv/gcn_conv.py

```diff
@@ -60,7 +60,8 @@ def gcn_norm(edge_index, edge_weight=None, num_nodes=None, improved=False,
         deg_inv_sqrt.masked_fill_(deg_inv_sqrt == float('inf'), 0.)
         adj_t = torch_sparse.mul(adj_t, deg_inv_sqrt.view(-1, 1))
         adj_t = torch_sparse.mul(adj_t, deg_inv_sqrt.view(1, -1))
-
+        adj_t = (1 - t) * torch.eye(
```

Comment:
IMO, it is better to place the residual part in the forward of `GCNConv` or a new module outside of `gcn_norm`.

Reply:
I'm open to it - but how would you suggest to implement this? The goal is to interpolate between the message passing operator (P) and the identity (I) like (1-t)I + tP, so it almost has to be done at this stage given the current implementation of `gcn_norm`.

Reply:
@EdisonLeeeee bumping this question, curious what you had in mind!

Reply:
Sorry for the delayed reply. The residual part can be performed after the (aggregated) output of `GCNConv`, i.e., `out = (1-t)*x + t*out`, WDYT?